ui-src/e2e/components/configurationComponent.e2e-spec.ts

```diff
@@ -189,4 +191,46 @@ describe("configuration component", () => {
       .expect(configPO.requiredFields.get(1).getText())
       .toEqual("Must be 100 or less");
   });
+
+  it("should check values of existing file are as expected", () => {
+    configPO.configFilePath.sendKeys(configPO.absolutePath);
+    configPO.xmlFileName.sendKeys("config");
+    configPO.getBtn.click();
+    expect(configPO.applicationName.getAttribute("value")).toEqual("Xtrac API");
+    expect(configPO.targetHost.getAttribute("value")).toEqual("localhost");
+    expect(configPO.targetPort.getAttribute("value")).toEqual("8080");
+    expect(configPO.testSuite.getAttribute("value")).toEqual("Default-1");
+    expect(configPO.testCaseDir.getAttribute("value")).toEqual(
+      "./definitions/testCases"
+    );
+    expect(configPO.baseStatsDir.getAttribute("value")).toEqual("./envStats");
+    expect(configPO.memoryEndpoint.getAttribute("value")).toEqual(
+      "/alt/debug/vars"
+    );
+    expect(configPO.reportsDir.getAttribute("value")).toEqual("./report");
+    expect(configPO.numIterations.getAttribute("value")).toEqual("1000");
+    expect(configPO.memoryVariance.getAttribute("value")).toEqual("15");
+    expect(configPO.serviceVariance.getAttribute("value")).toEqual("15");
+    expect(configPO.concurrentUsers.getAttribute("value")).toEqual("");
+    expect(configPO.requestDelay.getAttribute("value")).toEqual("5000");
+    expect(configPO.tpsFreq.getAttribute("value")).toEqual("30");
+    expect(configPO.rampDelay.getAttribute("value")).toEqual("15");
+    expect(configPO.rampUsers.getAttribute("value")).toEqual("5");
+  });
+  it("should update existing file", () => {
+    configPO.configFilePath.sendKeys(configPO.absolutePath);
+    configPO.xmlFileName.sendKeys("config");
+    configPO.getBtn.click();
+    configPO.numIterations.sendKeys(25);
+    configPO.btnUpdate.click();
+    expect(configPO.toastrMessage.getText()).toContain("Success!");
```

Comment:
You  should retrieve the file again to confirm that the update happened correctly.